monitoring/prometheus/egg-prometheus.json

```diff
@@ -37,6 +37,16 @@
             "user_editable": true,
             "rules": "required|string|max:20",
             "field_type": "text"
+        },
+        {
+            "name": "Data Save time in Days",
+            "description": "How long the data is being saved",
+            "env_variable": "DATA_SAVE_TIME",
+            "default_value": "365",
```

Comment:
by default this is 15, so let it be 15. also mention the units that it is using in the description

Reply:
considered and incorporated
thanks for the information